sunpy/net/tests/test_scraper.py

```diff
@@ -1,94 +1,173 @@
+import logging
 import datetime
+from urllib.error import URLError, HTTPError
 from unittest.mock import Mock, patch
 
 import pytest
-from dateutil.relativedelta import relativedelta
 
 from sunpy.data.test import rootdir
-from sunpy.net.scraper import Scraper, get_timerange_from_exdict
+from sunpy.extern import parse
+from sunpy.net.scraper import Scraper
+from sunpy.net.scraper_utils import get_timerange_from_exdict
 from sunpy.time import TimeRange, parse_time
 
-PATTERN_EXAMPLES = [
-    ('%b%y', relativedelta(months=1)),
-    ('%m%y', relativedelta(months=1)),
-    ('%H%d', relativedelta(hours=1)),
-    ('%y%b', relativedelta(months=1)),
-]
-
 
 def testDirectoryDatePattern():
-    s = Scraper('%Y/%m/%d/%Y%m%d_%H%M%S_59.fit.gz')
+    with pytest.deprecated_call():
```

Comment:
When you change to sunpydepcreationwarning, I am unsure if this will work. 

I would also like to check the message is the one we expect to be raised.